doc/sphinx/addendum/miscellaneous-extensions.rst

```diff
@@ -1,10 +1,5 @@
-.. _miscellaneousextensions:
-
-Miscellaneous extensions
-========================
-
 Program derivation
-------------------
+==================
 
 |Coq| comes with an extension called ``Derive``, which supports program
 derivation. Typically in the style of Bird and Meertens or derivations
```

Comment:
How is derivation different from extraction?

Reply:
The Derive plugin is a very unimportant feature that got its own chapter mostly by accident. The `Derive` command is very similar to the `Definition` command used in tactic mode. Extraction is a core feature that allows one to run Coq programs by means of translation to a real programming language like OCaml. It is very useful to people wanting verified OCaml or Haskell libraries.